cleverpush/src/main/java/com/cleverpush/stories/models/Widget.java

```diff
@@ -0,0 +1,133 @@
+
+package com.cleverpush.stories.models;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+public class Widget {
+
+    @SerializedName("_id")
+    @Expose
+    private String id;
+    @SerializedName("channel")
+    @Expose
+    private String channel;
+    @SerializedName("name")
+    @Expose
+    private String name;
+    @SerializedName("variant")
+    @Expose
+    private String variant;
+    @SerializedName("maxStoriesNumber")
+    @Expose
+    private Integer maxStoriesNumber;
+    @SerializedName("storyHeight")
+    @Expose
+    private Integer storyHeight;
+    @SerializedName("position")
+    @Expose
+    private String position;
+    @SerializedName("margin")
+    @Expose
+    private Integer margin;
+    @SerializedName("display")
+    @Expose
+    private String display;
+    @SerializedName("selectedStories")
+    @Expose
+    private List<Object> selectedStories = null;
+    @SerializedName("createdAt")
+    @Expose
+    private String createdAt;
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public String getChannel() {
+        return channel;
```

Comment:
please use this in return, as you want to reference to the class scope variable (make it more descriptive)